src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -59,6 +70,65 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
+        static bool FirstTimeUser()
+        {
+            // Check for the presence of HKCU:Software/NServiceBus
+            using (var regNsbRoot = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
+            {
+                if (regNsbRoot == null)
+                {
+                    //Check for the presence of HKCU:SOFTWARE\ParticularSoftware
+                    using (var regParticularRoot = Registry.CurrentUser.OpenSubKey(@"Software\ParticularSoftware"))
+                    {
+                        if (regParticularRoot == null)
+                        {
+                            return true;
+                        }
+                    }
+                }
+            }
+            return false;
+        }
+
+
+        private void TrackFirstTimeUsageEvent()
+        {
+            // Get the current version of NServiceBus that's being used
+            var assembly = Assembly.GetExecutingAssembly();
+            var version = assembly.GetName().Version.ToString();
+
+            if (assembly.Location != null)
+            {
+                version = FileVersionInfo.GetVersionInfo(assembly.Location).FileVersion;
+            }
+
+            try
+            {
+                var azureFuncUrl =
+                    "https://first-time-nuget-user.azurewebsites.net/api/TrackNewUser?code=1bkjzkbyeuf1ed46l87yom9529yu5ykzt81mjx65kgtmmv4pldiyxzvfprxf9s3xqxykf279cnmi";
+
+                var postData = $"{{\"version\":\"{version}\"}}";
+                var client = new HttpClient();
+                client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+
+                var request = new HttpRequestMessage(HttpMethod.Post, azureFuncUrl);
+                request.Content = new StringContent(postData, Encoding.UTF8, "application/json");
+                var response = client.SendAsync(request).GetAwaiter().GetResult();
+                response.EnsureSuccessStatusCode();
+            }
+            catch (Exception)
+            {
+                Logger.WarnFormat("Could not report first time user stat to GoogleAnalytics.");
+            }
+        }
+
+        void SetupFirstTimeRegistryKeys()
+        {
+            var regRoot = Registry.CurrentUser.CreateSubKey(@"Software\ParticularSoftware");
```

Comment:
This will blow up on azure web sites  see https://github.com/Particular/NServiceBus/issues/4143. Not sure how to deal with this other than ignoring it.